test/evptests.txt

```diff
@@ -2,6 +2,63 @@
 #aadcipher:key:iv:plaintext:ciphertext:aad:tag:0/1(decrypt/encrypt)
 #digest:::input:output
 
+# BLAKE2 tests, using same inputs as MD5
```

Comment:
Then add a comment above that test pointing to the RFC! :) It is useful to know where test vectors come from.